go-controller/pkg/cni/cni.go

```diff
@@ -93,24 +103,43 @@ func (pr *PodRequest) cmdAdd() *PodResult {
 		return nil
 	}
 
-	ovnAnnotation, ok := annotation["ovn"]
-	if !ok {
-		logrus.Errorf("failed to get ovn annotation from pod")
-		return nil
-	}
+	var ovnAnnotation string
```

Comment:
You don't need to declare it here, it is used only in the body of the first if block.

Reply:
done